testing/btest/opt/opt-skip-files.zeek

```diff
@@ -0,0 +1,16 @@
+# @TEST-REQUIRES: test "${ZEEK_USE_CPP}" != "1"
```

Comment:
Same as above re: combining these into one test.

Reply:
Here OTOH I could, since this one specifically avoids `-O gen-C++`, because it includes `-O ZAM` on the command line.